docs/usage/create-shoot-into-existing-aws-vpc.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Create a Shoot cluster into existing AWS VPC
```

Comment:
This is already described in https://github.com/gardener/gardener-extension-provider-aws/blob/master/docs/usage-as-end-user.md.

I am not sure whether we want to keep this guide, but if we want, it must not be in g/g but probably in https://github.com/gardener/gardener-extension-provider-aws/